src/commandMethod.py

```diff
@@ -0,0 +1,34 @@
+from subcommandMethod import *
+from helper import *
+
+def addCommand(subCommandType, courses):
+	subCommand = {
+		"course": addCourse,
+		"assessment": addAssessment,
+		"grade": addGrade,
+		"help": addHelp,
+		"-h": addHelp,
+	}
+	addSubCommand = subCommand.get(subCommandType, noSuchSubCommand)
+	addSubCommand(courses)
```

Comment:
do we need to make this more generalized?

Reply:
Looks good for now